src/createServer.js

```diff
@@ -100,15 +100,25 @@ function createServer(options) {
         "favicon": server.favicon
       };
 
+      function callback(err, response) {
```

Comment:
I think it would be better to give that function an other name, for example answerToPing